src/_index_test.js

```diff
@@ -54,23 +54,54 @@ describe("'synchronous' inspect", function() {
 		});
 	});
 
+    it("mocks isTTY value", function() {
+        // More inception!
+        stdout.inspectSync(function(output) {
+            var originalIsTTY = process.stdout.isTTY;
+            stdout.inspectSync({isTTY: !originalIsTTY}, function() {
+                assert.equal(process.stdout.isTTY, !originalIsTTY);
+            });
+            console.log("foo");
```

Comment:
Output assertions aren't necessary here because they've already been tested elsewhere. So "inception" nested `inspect` calls shouldn't be necessary either.